scalafix/input/src/main/scala/rsc/tests/RscCompat.scala

```diff
@@ -124,7 +125,8 @@ object RscCompat_Test {
     val compoundType6 = new A with B { def k: Int = ??? }
     val compoundType7 = ??? : A with (List[T] forSome { type T }) with B
 
-    val annType1 = ??? : C @ann
+    // TODO: Submit a ticket about this.
```

Comment:
Please submit a corresponding ticket and replace this comment with FIXME and a reference to that ticket.